install.rst

```diff
@@ -201,6 +202,51 @@ To install these, you can use:
   ``swig``, ``swig-python``, ``openssl``, ``bison``, and ``flex`` packages
   provide the required dependencies.
 
+* Windows
+
```

Comment:
Do you think we should add a note here to indicate that all of this is experimental? I know you put it in `NEWS` ... I'm just thinking that new folks might bump into this section more readily than the NEWS file.

Reply:
Added a section at the top about it.